keps/sig-api-machinery/20190612-crd-group-protection.md

```diff
@@ -0,0 +1,139 @@
+---
+title: k8s.io Group Protection
+authors:
+  - "@deads2k"
+owning-sig: sig-api-machinery
+participating-sigs:
+  - sig-api-machinery
+  - sig-architecture
+reviewers:
+  - "@sttts"
+  - "@jpbetz"
+  - "@liggitt"
+approvers:
+  - "@liggitt"
+  - "@sttts"
+editor:
+  name: "@deads2k"
```

Comment:
pretty this should be:
```suggestion
 - "@deads2k"
```
it's causing KEPs to fail to merge 
https://prow.k8s.io/?repo=kubernetes%2Fenhancements&type=batch
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/batch/pull-enhancements-verify/1153559205718790144
> keps/sig-api-machinery/20190612-crd-group-protection.md has an error: "error validating KEP metadata: \"editor\" must have a value"

kicking it out of the queue:
/lgtm cancel

Reply:
correction: it should be `editor: "@deads2k"`

Reply:
Sorry for the LGTM cancel, this did in fact let the others PRs merge, https://github.com/kubernetes/test-infra/issues/13551

Happy to re-LGTM when this is fixed.